.github/workflows/deploy.yml

```diff
@@ -0,0 +1,58 @@
+name: Deploy
```

Comment:
As there are workflows here that operate over CP and MP, could we change names of files (e.g. image-push-deploy.yml and deploy.yml) to refer to MP?